### ISSUE TITLE
When backfill is on, baritone should close doors and fence gates behind it

### ISSUE THREAD

**Issue by leijurv** [cabaletta/baritone#389]
only partially /s

**Comment by 0-x-2-2** [comment]
wtf it should always close doors and fence gates behind after it opens them that is really rude 😭

**Comment by ZeroMemes** [comment]
yeah wtf have u ever entered somebody's house and the door has been kept open wtf baritone 👎

**Comment by 0-x-2-2** [comment]
someone use baritone enter me house an baritone kept door open a zombie died me

**Comment by 0-x-2-2** [comment]
PWEASE FIX WEIJUWV AND BWADY

**Comment by usereewrjrjeejr** [comment]
Keep the doors open let everyone in matter creed, race, religion #FickTrump

**Comment by 0-x-2-2** [comment]
libtard detected

**Comment by EmeraldSnorlax** [comment]
> 
> 
> someone use baritone enter me house an baritone kept door open a zombie died me

wow, its as if killaura doesnt exist

**Comment by ZeroMemes** [comment]
killaura is not a baritone feature

**Comment by ZeroMemes** [comment]
Seriously though, this should be an option separate from backfill

**Comment by jari-kuipers** [comment]
This thread is ridiculous but the issue is very much real. Closing doors behind you isn't a feature and the Merge request above hasn't added this feature. Bumping this because as far as I know it's not implemented yet.
